src/sbt-test/skip-windows/caching/test

```diff
@@ -34,18 +34,19 @@ $ exec chmod 000 src/main/scala/Valid.scala
 > scalafix --check
 $ delete src/main/scala
 
-# a file patched during the last run should be cached
+# a file patched during the last run should not be cached as some rules might not have observed the latest content
 > set scalafixConfig := None
 $ mkdir src/test/scala
 $ copy-file files/ProcedureSyntax.scala src/test/scala/ToPatch.scala
 $ copy-file files/Valid.scala src/test/scala/Valid.scala
 # avoid rounding in mtime that could cause false negatives in `newer`
 $ sleep 1000
-> test:scalafix ProcedureSyntax
+> test:scalafix ProcedureSyntax DisableSyntax
```

Comment:
adding a second rule for exhibiting the scenario, even though behavior is the same no matter how many rules we have (but it might change one day)